app/controllers/favorites_controller.rb

```diff
@@ -19,7 +19,8 @@ def destroy
     if params[:id] == nil
       @favorites = session.clear
     else
-    @favorites.favorite_delete(params[:id])
+    pet = Pet.where(id: params[:id])
+    @favorites.favorite_delete(pet)
```

Comment:
This passes in a pet array (of one object) to account for the new favorite_delete method